jmeterUtils.go

```diff
@@ -224,7 +224,7 @@ func createDir(dir string) error {
 	if err := os.RemoveAll(dir); err != nil {
 		return err
 	}
-	return os.MkdirAll(dir, 0644)
+	return os.MkdirAll(dir, 0744)
```

Comment:
we should reflect over that change and why it is needed.
I'm assuming this has never caused a problem because it was run within a Docker container, but now it is problematic beause the tests run within github actions? :D

Reply:
To be able to read a directory in Linux the exactable bit has to be set for that directory, if the service runs as root this limitation is ignored. It's needed to run the service as non-root user or by the testing pipeline